website/sales/admin.py

```diff
@@ -0,0 +1,560 @@
+from functools import partial
```

Comment:
Maybe it would be nice to split these admin configs into multiple files for readability? :p